src/MessagingInput/MessagingInput.less

```diff
@@ -227,11 +227,6 @@
     min-height: 2rem;
     border-radius: 2rem;
 
-    .TextArea--input {
-      // Ensures that multi-line text doesn't spill outside the boundaries of the rounded text input
-      border-radius: 0.8rem;
```

Comment:
This is no longer necessary since the text was only spilling outside the boundaries of the input when there were multiple lines of text and the input was still fully rounded.